### PR TITLE
Add ImagineFun

### PR DESCRIPTION
**Pull request by camdencove**

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1065,5 +1065,16 @@
         "addresses": [
             "kingdomserver.be"
         ]
+    },
+       {
+        "id": "imaginefun",
+        "name": "ImagineFun",
+        "addresses": [
+            "mc.imaginefun.net",
+            "mc.imagineeringfun.net",
```
Comment (GrowlyX) [inline servers.json:1074]: You do not need to specify sub-domains

File: servers.json [hunk 2]
```diff
@@ -1065,5 +1065,16 @@
         "addresses": [
             "kingdomserver.be"
         ]
+    },
+       {
```
Comment (GrowlyX) [inline servers.json:1069]: This bracket is incorrectly formatted

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 1 above)
Comment (HelloImCamden) [reply servers.json]: Subdomains have been removed.

File: servers.json (hunk 2 above)
Comment (HelloImCamden) [reply servers.json]: Updated

**Review by GrowlyX — DISMISSED** [review]

**Review by GrowlyX — APPROVED** [review]

**Pull request merged**